.buildkite/pipeline.yml

```diff
@@ -18,7 +18,7 @@ steps:
     agents:
       queue: 't2medium'
 
-  - label: "style"
+  - label: ":python-black: style"
```

Comment:
(This and the 3.6 above probably didn't need to happen in this PR 😄 )

Reply:
I know, I couldn't resist!